frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield/ExpressionEditorTextfield.tsx

```diff
@@ -40,6 +45,50 @@ import {
 ace.config.set("basePath", "/assets/ui/");
 ace.config.set("useStrictCSP", true);
 
+export type SuggestionFooter = {
+  footer: true;
+  name: string;
+  icon: IconName;
+  href: string;
+};
+
+type SuggestWithFooters = {
+  suggestions: (Suggestion | SuggestionFooter)[];
+  helpText?: HelpText;
+};
+
+function suggestWithFooters(
```

Comment:
Adding the footer seemed misplaced in `metabase-lib/v1/expressions`, so I decided to do it here.

I wanted to keep the footers in the list of suggestions to avoid having to jump trough hoops to add the footer to the up/down arrow flow.